Libraries/src/Amazon.Lambda.KafkaEvents/KafkaEvent.cs

```diff
@@ -43,7 +43,7 @@ public class KafkaEventRecord
             /// <summary>
             /// The partition associated with the event record.
             /// </summary>
-            public string Partition { get; set; }
+            public int Partition { get; set; }
```

Comment:
How this change is verified in production?

Could you add to description?

Reply:
I got confirmation from the service team that it should be corrected. They have internal ticket to update the documentation. I spent 3 days trying to make MSK setup work, but no luck.